ethers-ffi/Makefile

```diff
@@ -40,37 +40,31 @@ android: android-setup android-build
 	cp $(TARGET_DIR)/i686-linux-android/release/$(SO) ${ANDROID_DEST}/x86/$(SO)
 	cp $(TARGET_DIR)/x86_64-linux-android/release/$(SO) ${ANDROID_DEST}/x86_64/$(SO)
 	cp $(TARGET_DIR)/aarch64-linux-android/release/$(SO) ${ANDROID_DEST}/arm64-v8a/$(SO)
-	cp $(TARGET_DIR)/arm-linux-androideabi/release/$(SO) ${ANDROID_DEST}/armeabi/$(SO)
+
 	cp $(TARGET_DIR)/armv7-linux-androideabi/release/$(SO) ${ANDROID_DEST}/armeabi-v7a/$(SO)
 
 aarch64-linux-android:
 	PATH=$(PATH):$(NDK_STANDALONE)/arm64/bin \
-	CC=$@-gcc \
-	CXX=$@-g++ \
-	cargo build $(CARGO_PARAMS) --target $@ --release --lib
-
-arm-linux-androideabi:
-	PATH=$(PATH):$(NDK_STANDALONE)/arm/bin \
-	CC=$@-gcc \
-	CXX=$@-g++ \
+	CC=$@-clang \
+	CXX=$@-clang++ \
```

Comment:
does this need to be gcc? can hopefully change it back and function as intended but wondering what's more clear for future usage.